tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/evaluation/DependencyEvaluationTest.java

```diff
@@ -42,7 +42,7 @@ public void setup() throws BallerinaTestException {
         debugTestRunner = new DebugTestRunner(testProjectName, testModuleFileName, true);
     }
 
-    @Test(description = "Test for expression evaluations against Ballerina lang library sources.")
+    @Test(enabled = false, description = "Test for expression evaluations against Ballerina lang library sources.")
```

Comment:
Why is this being disabled?

Reply:
There's an issue with the test related to expression evaluation https://github.com/ballerina-platform/ballerina-lang/issues/38646

Reply:
Please mention the issue (e.g., as a comment) in the test itself.